sotastream/__init__.py

```diff
@@ -1,10 +1,24 @@
+import os
 import sys
 
 sys.dont_write_bytecode = True
 
-from importlib import metadata
 
-__version__ = metadata.version(__package__)
+def get_version():
+    package_dir = os.path.dirname(__file__)
+    root_dir = os.path.abspath(os.path.join(package_dir, '..'))
+    pyproject_path = os.path.join(root_dir, 'pyproject.toml')
```

Comment:
Thanks for trying to fix this. 
We seem to have a problem here, -- this assumption of dir structure works when loaded directly from git repository but breaks when installed from pip.  Pip doesnt seem to copy pyproject.toml.

Reply:
I fixed this in the latest commit.